profiles/canopy_cloud_fabric.json

```diff
@@ -143,6 +179,27 @@
       ]
     },
 	{
+      "name": "javascript",
+      "runtime": "nodejs",
```

Comment:
You can add nodejs under runtimes. See https://github.com/stefan-kolb/paas-profiles#language